search/indexing_api.py

```diff
@@ -77,12 +96,13 @@ def create_document(doc_id, data):
         data (dict): Full ES document data
     """
     conn = get_conn(verify=True)
-    return conn.create(
-        index=settings.ELASTICSEARCH_INDEX,
-        doc_type=GLOBAL_DOC_TYPE,
-        body=data,
-        id=doc_id,
-    )
+    for alias in get_active_aliases():
+        return conn.create(
+            index=alias,
+            doc_type=GLOBAL_DOC_TYPE,
+            body=data,
+            id=doc_id,
+        )
```

Comment:
This returns immediately for the first item. Any items after that will be ignored

Reply:
Yeah that needs to be fixed... do we need the value returned from this function? If not I'm just going to remove that part